tests-e2e/cypress/integration/permissions_spec.ts

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/// <reference path="../support/index.d.ts" />
+
+import '@testing-library/cypress/add-commands';
+import {Channel} from 'mattermost-redux/types/channels';
+import {UserProfile} from 'mattermost-redux/types/users';
+
+import {httpStatusNotFound} from '../support/constants';
+
+describe('Test Area - Permissions', () => {
+    const fileHeader =
+    'Post Creation Time,User Id,User Email,User Type,User Name,Post Id,Parent Post Id,Post Message,Post Type';
+
+    before(() => {
+        // # Login as non-admin user.
+        cy.apiLogin('user-1');
+
+        // # Visit the default channel
+        cy.visit('/');
+
+        // # Set the Message Display to Standard
+        cy.setMessageDisplayToStandard();
```

Comment:
Non-blocking but since you're not testing the UI flow for changing display setting, it would be nice to do it via API. See `cy.apiSaveMessageDisplayPreference` in the webapp for reference.

Reply:
Done in https://github.com/mattermost/mattermost-plugin-channel-export/pull/14/commits/c3e900a22ca7df95fa0480d57cb279aa77af61c1.